### PR TITLE
Improve script and README

### PR DESCRIPTION
**Pull request by 0xjei**
This PR use `figlet`, `chalk` and `node-emoji` packages to style the script. Then, a up-to-date README version is provided.

### CODE REVIEW

File: src/bandada-sdk.ts [hunk 1]
```diff
@@ -1,33 +1,53 @@
 import { ApiSdk } from "@bandada/api-sdk"
+import figlet from "figlet"
+import chalk from "chalk"
+import * as emoji from "node-emoji"
 import "dotenv/config"
 
+// Theme.
+const log = console.log
```
Comment (vplasencia) [inline src/bandada-sdk.ts:8]:
```suggestion
const { log } = console
```

There was an ESLint error that can be fixed with this.

**Review by vplasencia — CHANGES_REQUESTED** [review]
Everything looks great! Very beautiful 🤩

Thanks for the great work 🔥

I just left one comment.

**Comment by vplasencia** [comment]
Which version of yarn are you using? I'm getting a different `yarn.lock` file.

**Comment by 0xjei** [comment]
> Which version of yarn are you using? I'm getting a different `yarn.lock` file.

The repository did not specify which version to use, so my system defaulted to the legacy version 1.22.19. I will update it now! Thank you

**Comment by 0xjei** [comment]
@vplasencia updated to latest yarn stable version (4.1.0)

File: .yarnrc.yml [hunk 2]
```diff
@@ -1 +1,3 @@
 nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.1.0.cjs
```
Comment (vplasencia) [inline .yarnrc.yml:3]: We don't have this folder in the project. I think we can create it or just leave the version in the package.json file.

**Review by vplasencia — APPROVED** [review]

**Pull request merged**